.env.example

```
@@ -0,0 +1,99 @@
+
+# ------------------------------------------------------------------------------
+# Database Configuration (PostgreSQL for MemU)
+# ------------------------------------------------------------------------------
+# For production: Use PostgreSQL with pgvector extension
+# For development: Can use in-memory mode (set MEMU_MODE=inmemory)
+
+# Database credentials (used by docker-compose)
+POSTGRES_USER=postgres
+# WARNING: Use a strong, unique password in production! Generate with:
+# python -c "import secrets; print(secrets.token_urlsafe(24))"
+POSTGRES_PASSWORD=CHANGE_THIS_PASSWORD
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)